lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -50,29 +53,52 @@ sub new {
             die "$@ \n" if $@;
             $self->{db} = $backend_module->new();
         };
-        die "$@ \n" if $@;
+        if ($@) {
+            warn "Failed to initialize the database backend module: [$@] \n";
+            die "Failed to initialize the database backend module \n" if $@;
```

Comment:
I feel that the message to the API client is too long and contains too much information. I suggest "INTERNAL SYSTEM ERROR" or something.

Reply:
@matsduf Please see my comment about (standard) JSON-RPC error messages above.

Reply:
> @matsduf Please see my comment about (standard) JSON-RPC error messages above.

"Internal error" à la RPC is fine.

Reply:
Fixed in commit 416b48b